comparisons/forms/color_picker/a11y.txt

```diff
@@ -0,0 +1 @@
+Usage: Safe for production.
```

Comment:
Probably want to add a label to the input and submit button before saying it's safe for production (I'm just looking at the live site, not sure if you've already done this).